src/format/command.rs

```diff
@@ -96,8 +96,23 @@ pub fn read_and_format_paths(files: Vec<String>) -> Result<Vec<Formatted>, Error
         })?;
 
         if path.is_dir() {
-            for path in file::gleam_files(&path).into_iter() {
-                formatted_files.push(format_file(path)?);
+            let mut gitignore_file = path.clone();
+            gitignore_file.push(".gitignore");
```

Comment:
Can we use `join` rather than `clone` + `push`?

Reply:
Definitely! I didn't know .join was a thing!!